recipes/pyaaf2/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "pyaaf2" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 160d3c26c7cfef7176d0bdb0e55772156570435982c3abfa415e89639f76e71b
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
```

Comment:
When you make this noarch, please specify a lower bound for python, e.g. `python >=3.6`